Source: written by Paul Seidler, based on webkit-gtk-1.3
Upstream: no
Reason: need gobject-introspection-0.9 compatibility for libsoup[gobject-introspection]

--- a/WebKit/gtk/JSCore-1.0.gir
+++ b/WebKit/gtk/JSCore-1.0.gir
@@ -1,12 +1,14 @@
 <?xml version="1.0"?>
-<repository version="1.0"
+<repository version="1.2"
             xmlns="http://www.gtk.org/introspection/core/1.0"
             xmlns:c="http://www.gtk.org/introspection/c/1.0">
   <namespace name="JSCore" version="1.0" shared-library="webkit-1.0">
-    <alias name="GlobalContextRef"
-           target="none"
-           c:type="JSGlobalContextRef"/>
-    <alias name="ObjectRef" target="none" c:type="JSObjectRef"/>
+    <alias name="GlobalContextRef" target="none">
+      <type name="JSGlobalContextRef" c:type="JSGlobalContextRef"/>
+    </alias>
+    <alias name="ObjectRef" target="none">
+      <type name="JSObjectRef" c:type="JSObjectRef"/>
+    </alias>
     <function name="EvaluateScript" c:identifier="JSEvaluateScript">
       <return-value transfer-ownership="none">
         <type name="none" c:type="void"/>
--- a/configure.ac
+++ b/configure.ac
@@ -630,7 +630,7 @@
 GIRTYPELIBDIR=

 if test "$enable_introspection" = "yes"; then
-  GOBJECT_INTROSPECTION_REQUIRED=0.6.2
+  GOBJECT_INTROSPECTION_REQUIRED=0.6.15
   PKG_CHECK_MODULES([INTROSPECTION],[gobject-introspection-1.0 >= $GOBJECT_INTROSPECTION_REQUIRED])

   G_IR_SCANNER="$($PKG_CONFIG --variable=g_ir_scanner gobject-introspection-1.0)"
